src/client/src/containers/SelectFrameworks/index.tsx

```diff
@@ -43,7 +43,7 @@ class SelectFrameworks extends React.Component<Props> {
 
   render() {
     return (
-      <div className={styles.container}>
```

Comment:
just wondering, why was this removed?

Reply:
it made the select frameworks page inconsistent with all the other pages - the only noticeable difference .container made was additional padding that made it look worse LOL